ee/agent/timemachine/timemachine_darwin.go

```diff
@@ -0,0 +1,36 @@
+//go:build darwin
+// +build darwin
+
+package timemachine
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/kolide/launcher/ee/agent/types"
+	"github.com/kolide/launcher/ee/allowedcmd"
+)
+
+// ExcludeLauncherDB adds the launcher db to the time machine exclusions for
+// darwin and is noop for other oses
+func ExcludeLauncherDB(ctx context.Context, k types.Knapsack) {
+	dbPath := k.BboltDB().Path()
+	cmd, err := allowedcmd.Tmutil(ctx, "addexclusion", dbPath)
+	if err != nil {
+		k.Slogger().Log(ctx, slog.LevelError,
+			"failed to add launcher db to time machine exclusions",
```

Comment:
Usually this error would be tmutil not being found --

```suggestion
			"could not create tmutil command to add launcher db to time machine exclusions",
```

Reply:
updated